Source/WebKit/WebProcess/Network/WebLoaderStrategy.cpp

```diff
@@ -188,7 +188,7 @@ void WebLoaderStrategy::scheduleLoad(ResourceLoader& resourceLoader, CachedResou
         trackingParameters.webPageProxyID = valueOrDefault(webFrameLoaderClient->webPageProxyID());
     else if (is<RemoteWorkerFrameLoaderClient>(frameLoaderClient))
         trackingParameters.webPageProxyID = downcast<RemoteWorkerFrameLoaderClient>(frameLoaderClient).webPageProxyID();
-    trackingParameters.pageID = valueOrDefault(frameLoaderClient.pageID());
+    trackingParameters.pageID = valueOrDefault(resourceLoader.frame()->pageID());
```

Comment:
maybe null check this

Reply:
Then it would crash later during IPC because we cannot send an invalid ObjectIdentifier over IPC. As I remember it, pageID is not optional.

Reply:
Also, we already asserted above that `resourceLoader.frameLoader()` wasn't null, which means that `resourceLoader.frame()` cannot be null either.

Reply:
Switching to `frameLoader->frame().pageID()` to make it clearer frame cannot be null.